che-core-git-impl-native/src/main/java/org/eclipse/che/git/impl/nativegit/commands/LogCommand.java

```diff
@@ -46,6 +47,9 @@ public LogCommand(File place) {
         if (count > 0) {
             commandLine.add("-" + count);
         }
+        if (fileFilter != null && !fileFilter.isEmpty()) {
```

Comment:
Please take a look implementation of CommandLine https://github.com/codenvy/che-core/blob/master/platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/util/CommandLine.java#L62-L67
I think we can miss these checkings